.github/workflows/build.yml

```diff
@@ -116,11 +149,8 @@ jobs:
         security import certificate.p12 -k build.keychain -P $MACOS_CERTIFICATE_PWD -T /usr/bin/codesign
         security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k Techminohaowan build.keychain
         /usr/bin/codesign --force --deep -s 79B81FC5EA155243C973B5417B0996501F00EF55 ./Techmino.app -v
-    - name: Pack Techmino
-      run: |
-        zip -r -y Techmino-Packed.app.zip Techmino.app
-    - name: Artifact
-      uses: actions/upload-artifact@v1.0.0
+    - name: Upload
```

Comment:
The uploading of folders breaks symlinks, which are crucial to the integrity of App Bundles.

I suggest we stick with `zip -r -y`, which explicitly keeps the symlinks inside the folders. When an OS X user opens the zip file, it will automatically be unpacked twice, and that's acceptable.

Reply:
This is even worse than expected, because github doesn't allow uploading folders at all. These lines of changes should be reverted.